lib/ui/components/weather_image.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:flutter_training/data/provider/yumemi_weather_provider.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class WeatherImage extends ConsumerWidget {
+  const WeatherImage({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final weather = ref.watch(yumemiWeatherProvider);
+
+    switch (weather) {
+      case Weather.sunny:
+        return SvgPicture.asset(
+          'assets/images/sunny.svg',
+          semanticsLabel: 'sunny',
+        );
+      case Weather.cloudy:
+        return SvgPicture.asset(
+          'assets/images/cloudy.svg',
+          semanticsLabel: 'cloudy',
+        );
+      case Weather.rainy:
+        return SvgPicture.asset(
+          'assets/images/rainy.svg',
+          semanticsLabel: 'rainy',
+        );
+      default:
```

Comment:
[imo]
できる限り、default は使わないことをおすすめします。
理由としては、後から enum の種類が追加になったときに、修正漏れが発生する可能性があり、保守性を低下させていまいます。
この場合は 以下のようにするとよさそうです！

```suggestion
      case null:
```

Reply:
@blendthink 

細かく見ていただきありがとうございます🙇‍♂️
d9622c7 こちらで対応しましたのでご確認よろしくお願いします！